### PR TITLE
Adding tooling section

### PR DESCRIPTION
**Pull request by sublibra**

### CODE REVIEW

**Comment by qlikossbuild** [comment]
Deploy preview for *core-website* ready!

Built with commit b33e129f3baf653cda353f19e0b47b24d6c08d4a

https://deploy-preview-447--core-website.netlify.com

File: docs/tooling/corectl.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Corectl
+
+Qlik Core Command (corectl) is a tool that delivers a command line interface (CLI) for the Qlik Associative Engine. With
```
Comment (peol) [inline docs/tooling/corectl.md]: `Qlik Core Control` (I'm guessing `ctl` is for `control` and not `command`)?

File: mkdocs.yml [hunk 2]
```diff
@@ -67,6 +67,10 @@ nav:
       - Local Files: tutorials/data-loading/local-files.md
       - Remote Files: tutorials/data-loading/remote-files.md
       - Databases: tutorials/data-loading/databases.md
+  - Tooling:
+    - Overview: tooling/overview.md
+    - corectl: tooling/corectl.md
+    - Catwalk: tooling/catwalk.md
```
Comment (peol) [inline mkdocs.yml]: `Catwalk` or `catwalk`? I've only seen `catwalk` elsewhere

Comment (sublibra) [reply mkdocs.yml]: It should be ```catwalk```

**Pull request merged**